packages/web/src/components/gcds-date-modified/gcds-date-modified.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Element, Host, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-date-modified',
+  styleUrl: 'gcds-date-modified.css',
+  shadow: false,
+  scoped: true
+})
+export class GcdsDateModified {
+  @Element() el: HTMLElement;
+
+  private lang: string;
+
+  async componentWillLoad() {
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+  }
+
+  render() {
+    return (
+      <Host>
```

Comment:
Please add a `gcds-` prefixed class to the host or the `<dl>` tag to follow the props/classes consistency.